bundle/regal/ast.rego

```diff
@@ -136,6 +141,32 @@ find_vars(node) := [var |
 	some var in _find_vars(path, value, regal.last(path))
 ]
 
+# METADATA
+# description: |
+#  finds all vars declared in `rule` *before* the `location` provided
+#  note: this isn't 100% accurate, as it doesn't take into account `=`
+#  assignments / unification, but it's likely good enough since other rules
+#  recommend against those
```

Comment:
Yep! "Luckily" we don't yet complain about `=` for [var assignments](https://github.com/StyraInc/regal/issues/3).

But that should be easier to implement now that we have the plumbing in place for determining whether `=` is used for assignment or comparison (i.e. whether the vars are inputs or outputs)... so that would be up next I guess.